.github/workflows/no-response.yml

```diff
@@ -12,6 +12,9 @@ on:
 jobs:
   noResponse:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
I think these changes broke the workflow:
- issues with no response aren't closed automatically after 7 days - https://github.com/mui/mui-x/issues/6447
- the label is not automatically removed after issue author's response - https://github.com/mui/mui-x/issues/6453